lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -153,7 +164,7 @@ function M.select_mv_cmd(from, to, cwd)
     return {
       cmd = "cmd",
       opts = {
-        args = { "/C", "move", "/Y", from, to },
+        args = { "/C", "move", "/Y", cmdpath(from), cmdpath(to) },
```

Comment:
@tajtiattila can this normalization happen at the beginning of the functions? Otherwise it is easy to forget to add `cmdpath` when more commands are added.

Reply:
Hmm, but maybe is the `cmdpath` for `"cmd"` invocation also a reminder to remember that. I'm fine with either solution that you prefer!

Reply:
I believe someone adding new commands using `cmd.exe` will stumble upon `cmdpath` and realize it is needed for paths.

I thought about better ways to do it, but I couldn't find any. Perhaps commands could use powershell, but I don't know much about it. I always use bash, which is why I prefer using `shellslash` in (Neo)Vim.